next/src/app/blog/[slug]/page.tsx

```diff
@@ -0,0 +1,123 @@
+import { notFound } from 'next/navigation';
+import Hero from '@/components/_article/hero';
+import AuthorComponent from '@/components/_global/authorComponent';
+import ContentSection from '@/components/_global/contentComponent/contentComponent';
+import SEO from '@/global/Seo';
+import { type BlogPostPage } from '@/types/_pages/BlogPostPage';
+import { sanityFetch } from '@/utils/sanity-client';
+
+export async function generateStaticParams() {
+  const allBlogPosts = await getAllBlogPosts();
+  return allBlogPosts.map(({ slug }) => ({
+    slug: slug.current,
+  }));
+}
+
+export async function generateMetadata({ params: { slug } }: { params: { slug: string } }) {
+  const { seo } = await getMetadata(slug);
+  return SEO({
+    title: seo?.title,
+    description: seo?.description,
+    url: '/co-robie',
```

Comment:
Wrong path for SEO provided. Take a look globally for SEO.

Reply:
rest will be set in dev branch